src/branches.cpp

```diff
@@ -269,6 +272,10 @@ namespace l
   int
   erase_begin(Branches::Impl *branches_)
   {
+    if (branches_->size() <= 1) {
```

Comment:
If you don't mind updating the code to use a more GNU style for braces and such. And I'm fine with something like 

```
if(branches_->size() <= 1)
  return -ENOTSUP;
```

Reply:
OK, updated the code to better match the style